acceptance/lib/puppet/acceptance/environment_utils.rb

```diff
@@ -0,0 +1,44 @@
+module Puppet
+  module Acceptance
+    module EnvironmentUtils
+
+     # generate a random string of 6 letters and numbers.  NOT secure
+      def random_string
+        [*('a'..'z'),*('0'..'9')].shuffle[0,8].join
+      end
+      private :random_string
+
+      # if the first test to call this has changed the environmentpath, this will cause trouble
+      #   maybe not the best idea to memoize this?
```

Comment:
It does seem potentially troublesome to memorise the environmentpath. As a test developer; I would (like to) assume that helper lib methods are stateless.